packages/spans/package.json

```diff
@@ -2,40 +2,40 @@
   "name": "@embrace-io/react-native-spans",
   "version": "4.1.2",
   "description": "Embrace’s Performance Tracing solution gives you complete visibility into any customized operation you’d like to track, enabling you to identify, prioritize, and resolve any performance issue",
-  "author": {
-    "name": "Embrace.io",
-    "email": "truenorth@embrace.io"
+  "author": "Embrace <support@embrace.io> (https://embrace.io/)",
+  "bugs": {
+    "url": "https://github.com/embrace-io/embrace-react-native-sdk/issues",
+    "email": "support@embrace.io"
   },
-  "homepage": "https://embrace.io/",
-  "license": "ISC",
-  "main": "lib/src/index.js",
-  "typings": "lib/src/index.d.ts",
+  "homepage": "https://github.com/embrace-io/embrace-react-native-sdk",
```

Comment:
I think "react-native": ">=0.56.0" is missing for spans

Reply:
added this to the yarn constraints so now each package.json should have it, is it fine that everything in packages/* specifies it now?

Reply:
nono its fine